swift/ios-shoppe-demo/AppDelegate.swift

```diff
@@ -13,6 +13,7 @@ import FullStory
 class AppDelegate: UIResponder, UIApplicationDelegate, FSDelegate {
 
     func fullstoryDidStartSession(_ sessionUrl: String) {
+        print(sessionUrl)
```

Comment:
Does print help or is it possibly leftover from debugging the app?

Reply:
left over debugging your right I will fix this :)